pootle/apps/import_export/utils.py

```diff
@@ -12,35 +12,32 @@
 from django.utils.translation import ugettext as _
 
 from pootle_store.models import Store
+from pootle_statistics.models import SubmissionTypes
 
+from .exceptions import (UnsupportedFiletypeError, MissingPootlePathError,
+                         MissingPootleRevError, FileImportError,
+                         ERR_UNSUPPORTED_FILETYPE, ERR_MISSING_POOTLE_PATH,
+                         ERR_MISSING_POOTLE_REV, ERR_FILE_IMPORT)
```

Comment:
According to our styleguide http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#imports the constants go first.